src/main/java/com/example/EventForgeFrontend/controller/OrganisationController.java

```diff
@@ -36,13 +33,10 @@ public String  showAllOrganisationForUnauthorized(@RequestParam(value = "name" ,
         return "allOrganisations";
     }
 
-    @GetMapping("/{orgName}/get-events/{id}")
-    public String showOrganisationEventsById(@PathVariable("orgName")String orgName ,@PathVariable("id")Long id ,Model model){
-      ResponseEntity<EventResponseContainer> result=  unauthorizeApiClient.showOrgEvents(orgName ,id);
-      List<OneTimeEventResponse> oneTimeEvens = result.getBody().getOneTimeEvents();
-      List<RecurrenceEventResponse> recurrenceEvents = result.getBody().getRecurrenceEvents();
-      model.addAttribute("oneTimeEvents" ,oneTimeEvens);
-      model.addAttribute("recurrenceEvents" , recurrenceEvents);
+    @GetMapping("/{orgName}/get-events")
+    public String showOrganisationEventsById(@PathVariable("orgName")String orgName ,Model model){
```

Comment:
it looks like the method returns all events by organisation's name, not by organisation's id